doc/uno-toolkit-migration.md

```diff
@@ -0,0 +1,26 @@
+---
+uid: Toolkit.Migration
+---
+
+# Upgrading Uno Toolkit
+
+## Upgrading to Uno Toolkit 7.1
+
+1. Bump to Uno 6.0
+
+    Version 7.1 of Uno Toolkit now requires `Uno 6.0`.
+
+2. Switch from `ReturnType` to `InputReturnType`
+
+    The `InputExtensions` helper has been updated to use the built-in `InputReturnType` enum (provided by Uno v6+) instead of the legacy `ReturnType` enum.
```

Comment:
Reflecting back on this, couldn't we keep the original `ReturnType` enum as the type for the ReturnType and convert it to the `InputReturnType` enum behind the scenes? That way we wouldn't have a breaking change here

cc: @Xiaoy312

Reply:
Yeah, unless you are setting from code-behind I guess

Reply:
I feel like it's still to be considered a breaking change no? Client code would need to be recompiled as the code generated from XAML would break